besspin/cyberPhys/canlib/lib/canspecs.js

```diff
@@ -0,0 +1,157 @@
+/*
+* Cyberphys CAN Frames Specification
+* Project: SSITH CyberPhysical Demonstrator
+* Name: lib/canspecs.js
+* Author: Steven Osborn <steven@lolsborn.com>, Kristofer Dobelstein, Ethan Lew <elew@galois.com>
+* Michal Podhradsky <mpodhradsky@galois.com>
+* Date: 07 June 2021
+* This file was created by BESSPIN-Tool-Suite/besspin/cyberPhys/canlib/make_can_spec.py
+*/
+
+// Name: gear
+// Units: N/A
+// Type: uint8_t
+// Description: selected gear
+const CAN_ID_GEAR = 0XAAF01000
+const CAN_FORMAT_GEAR = "!B"
```

Comment:
Could probably leave out the FORMAT variables in the javascript file as this is a python thing and JS doesn't understand it.

Reply:
Good point, fixed